src/core_plugins/metrics/public/components/lib/get_axis_label_string.js

```diff
@@ -0,0 +1,25 @@
+import { relativeOptions } from '../../../../../ui/public/timepicker/relative_options';
+import _ from 'lodash';
+import moment from 'moment';
+const unitLookup = {
+  s: 'seconds',
+  m: 'minutes',
+  h: 'hours',
+  d: 'days',
+  w: 'weeks',
+  M: 'months',
+  y: 'years'
+};
+export function getAxisLabelString(interval) {
+  const units = _.pluck(_.clone(relativeOptions).reverse(), 'value')
```

Comment:
There's some duplication of this. `units` is a global constant, and we also use it in `parse_relative_parts.js`.

I would extract the `units-array` and the `unitLookup`-map to a different module, and pull it in where needed. E.g. add the units-array and the lookup table to the `time_units` module.